Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/IndexService.java

```diff
@@ -6,13 +6,14 @@
 import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
Line number 6 doesn't show as a change in the file. But I have formatted the entire file since few lines were using tab spacing.